SPECS/php-pecl-apcu/php-pecl-apcu.spec

```diff
@@ -0,0 +1,456 @@
+# Fedora spec file for php-pecl-apcu
+#
+# Copyright (c) 2013-2024 Remi Collet
+# License: CC-BY-SA-4.0
+# http://creativecommons.org/licenses/by-sa/4.0/
+#
+# Please, preserve the changelog entries
+#
+
+%global pecl_name  apcu
+%global with_zts   0%{?__ztsphp:1}
+%global ini_name   40-%{pecl_name}.ini
+%global sources    %{pecl_name}-%{version}
+
+Name:           php-pecl-apcu
+Summary:        APC User Cache
+Version:        5.1.23
```

Comment:
Superseeds:
- https://github.com/microsoft/azurelinux/pull/10414
- https://github.com/microsoft/azurelinux/pull/10413
- https://github.com/microsoft/azurelinux/pull/10319

Can we please for subsequent PRs update initial PR instead of closing and creating a new one.
It helps to keep the review audit.